src/dlgConnectionProfiles.cpp

```diff
@@ -128,6 +131,9 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
         Q_ASSERT_X(!cursor.isNull(), "dlgConnectionProfiles::dlgConnectionProfiles(...)",
                    "CONNECT_PROFILE_ICON text marker not found in welcome_message text for when icons are shown on dialogue buttons");
         cursor.removeSelectedText();
+        QImage image_load(QPixmap(icon_connect.pixmap(offline_button->iconSize())).toImage());
```

Comment:
Does this mean I can remove the line?
I'd be happy to.
My system does not show icons in this context, so it's hard for me to know what it's doing. I just copied what the connect button did and figured if the Windows users got icons, then great.

Reply:
:neckbeard: To simulate it you can rename the \<user's home directory>`/.config/mudlet` directory to something else *temporarily* then when you fire up Mudlet you'll get something like:

+ With icons on buttons:
![Screenshot_20190601_201019](https://user-images.githubusercontent.com/6163092/58752604-d9355000-84a9-11e9-8b75-b75bb147e55a.png)
+ Without icons on buttons:
![Screenshot_20190601_201302](https://user-images.githubusercontent.com/6163092/58752608-ebaf8980-84a9-11e9-807a-f986c96fe3cd.png)

Settings for the icons on buttons *on KDE in GNU/Linux* is via the System's setting options for "Appearance" - "Application Style" - "Widget Style" - "Fine Tuning"
![Screenshot_20190601_201411](https://user-images.githubusercontent.com/6163092/58752623-4ea12080-84aa-11e9-9c5a-dd953832eb29.png)

After debugging you can delete the ***newly created*** \<user's home directory>`/.config/mudlet` directory and then rename the temporarily renamed one back to that.

*However this has just shown me that there is a bug and the wrong icon is being shown for the "Connect" button as it is the default Qt one and not the one from the Mudlet resources file that is integral to the application executable file -  so something has gone wrong **there**.*

Reply:
Yikes - I should have emphasised - THIS IS SOMETHING OF A BUG - as it is it means that the icon for the "Connect" button in the "Welcome message" ought to be replaced by the image of a hard hat on the mudlet arch and not the (intended) circular star icon or the (actual) tick icon that is the Qt default for an "AcceptRole" button.

However, this is mitigated because the Welcome message will still be showing the Qt default icon for the ApplyRole button (a monochrome tick in the top one of the above screen shots) and not the correct "Connect" icon - it thus may be a *little* less obvious (the wording will still differentiate them) whether the user should click on the `"Connect"` or the `"Load"` button.